e2e/cypress/integration/api/iam/02_policies_api.spec.ts

```diff
@@ -368,12 +370,30 @@ describeIfIAMV2('policies API', () => {
             method: 'PUT',
             url: `/apis/iam/v2beta/policies/${policyID}`,
             failOnStatusCode: false,
-            body: policyWithProjects(policyID, [project1.id, project2.id], statementProjects)
+            body: policyWithProjects(policyID, [project1.id], statementProjects)
         }).then((response) => {
             expect(response.status).to.equal(403);
         });
       });
 
+      it('non-admin with project1 assignment access can update ' +
+      'a policy with project1 to have no projects', () => {
+        cy.request({ ...defaultAdminReq,
+            method: 'POST',
+            body: policyWithProjects(policyID, [project1.id], statementProjects)
+        }).then((response) => {
+            expect(response.body.policy.projects).to.have.length(1);
+        });
+
+        cy.request({ ...defaultNonAdminReq,
+            method: 'PUT',
+            url: `/apis/iam/v2beta/policies/${policyID}`,
+            body: policyWithProjects(policyID, [], statementProjects)
+        }).then((response) => {
+            expect(response.body.policy.projects).to.have.length(0);
+        });
+      });
```

Comment:
1. This test is now a duplicate of the one on L323.

2. I am puzzled though; I thought that this would be disallowed...?

Reply:
This case is allowed, although it is a tricky edge case. We decided to allow updates to unassigned for the time being.

Reply:
Good catch-- I'll modify the other to leave another project behind